ONSdigital/census-rm-case-service
# migration of rm-case-service for census-rm-case-service

Why?
New fork for census-rm

What's changed
removed manifest and jenkins, renamed in pom, dockerfile and .travis

how to test:
build image locally and check still works

links:
https://trello.com/c/6arZY5kG/603-fork-rm-case-service
